src/eva/vision/data/wsi/base.py

```diff
@@ -0,0 +1,51 @@
+import abc
+from typing import Any, List, Tuple
+
+import numpy as np
+
+
+class Wsi(abc.ABC):
```

Comment:
let's add a docstring here (maybe just `"""Base class for holding WSI data"""`)

Reply:
done

Reply:
done